packages/api/src/command/hie/unlink-patient-from-organization.ts

```diff
@@ -242,13 +251,16 @@ async function findAndRemoveConsolidatedDocumentFromS3(
 ): Promise<void> {
   const dryRunMsg = getDryRunPrefix(dryRun);
   try {
-    const consolidatedPrefix = createConsolidatedSnapshotFileNameWithNoExtension(cxId, patientId);
-    const medicalRecordsPrefix = createMRSummaryFileNameWithNoExtension(cxId, patientId);
+    const consolidatedPrefix = createConsolidatedSnapshotFileNameWithSuffix(cxId, patientId);
```

Comment:
nit: can we name it `consolidatedSnapshotPrefix` to minimize confusion w/ the `consolidatedDataPrefix`, please?